src/lib/utilities/request-from-api.ts

```diff
@@ -159,6 +182,27 @@ const withBearerToken = async (
   return headers;
 };
 
+const withIdToken = (
+  headers: HeadersInit,
+  idToken: string,
+  isBrowser = browser,
+): HeadersInit => {
+  // At this point in the code path, headers will always be set.
+  /* c8 ignore next */
+  if (!headers) headers = {};
+  if (!isBrowser) return headers;
+
+  try {
+    if (idToken) {
+      headers['Authorization-Extras'] = idToken;
+    }
```

Comment:
Why does this need to be in a try catch?

Reply:
oops